config/settings.yml

```diff
@@ -42,6 +42,7 @@ discourse_calendar:
   close_to_working_day_hours_extension:
     default: 2
     client: true
+  calendar_automatic_holidays_enabled: true
```

Comment:
Not sure this is right...should be:

```ruby
calendar_automatic_holidays_enabled:
  default: true
```

Reply:
Maybe I am wrong and we allow that other syntax?

Reply:
yeah the other syntax is only required when you are using client: true... I prefer the more concise one where possible

Reply:
Right, carry on then!